lib/Account.php

```diff
@@ -120,7 +120,7 @@ public function deauthorize($clientId = null, $opts = null)
      * @param array|null $params
      * @param array|string|null $opts
      *
-     * @return ExternalAccount
+     * @return ApiResource
      */
     public static function createExternalAccount($id, $params = null, $opts = null)
```

Comment:
Should re remove those methods explicitly in favor of `$account->external_accounts->create()` instead? This avoids the confusion and we would automatically return the right class based on the type of external accounts

Reply:
No, these methods let users manipulate external accounts without having to retrieve the account first (like other nested resource methods). Also, to clarify, these methods _will_ return the correct type -- though of course we cannot know what this type will be in the PHPDoc.